compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/StreamingCodeDesugar.java

```diff
@@ -93,6 +93,7 @@
 import org.wso2.ballerinalang.compiler.util.FieldKind;
 import org.wso2.ballerinalang.compiler.util.Name;
 import org.wso2.ballerinalang.compiler.util.Names;
+import org.wso2.ballerinalang.compiler.util.TypeTags;
```

Comment:
Do we need this import? there no other changes in this class.

Reply:
I merged the latest master changes to branch and some how this import is missing. Need to add this to since some earlier changes in refactor branch need this, else build will be failed.